test/cluster/test_tablets_merge.py

```
@@ -652,3 +652,111 @@ async def finished_merge():
+    cmdline = [
+        '--logger-log-level', 'compaction_manager=info',
+        '--logger-log-level', 'compaction=info',
+    ]
+    server = await manager.server_add(cmdline=cmdline, config=cfg)
+
+    cql = manager.get_cql()
+    await manager.disable_tablet_balancing()
+
+    initial_tablets = 2
+
+    async with new_test_keyspace(manager, f"WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}") as ks:
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)